test/functional/admin/editions_controller_test.rb

```diff
@@ -81,6 +81,25 @@ class Admin::EditionsControllerTest < ActionController::TestCase
     assert_redirected_to edit_admin_publication_path(draft_edition.reload)
   end
 
+  test "failing to revise an edition should surface any ActiveRecord errors" do
+    published_consultation = create(:consultation_with_public_feedback_html_attachment)
+    public_feedback = published_consultation.public_feedback
+    public_feedback.summary = <<~MARKDOWN
+      Here is a simple footnote[^1]. With some additional text after it.
+
+      [^1]: Footnote which is not allowed by consultations.
+    MARKDOWN
```

Comment:
NB, I did try exposing the [SafeHtmlValidator](https://github.com/alphagov/whitehall/blob/59b7250a02139f36725fcd495a4df92f92de0f66/app/models/consultation_response.rb#L10) using:

```ruby
public_feedback.summary = "<script>alert('not allowed')</script>"
```

...but it validated fine, despite it failing when checking the console directly:

```
Govspeak::HtmlValidator.new("<script>alert('not allowed')</script>").valid?
=> false
```

That's the [same code as appears in the validator](https://github.com/alphagov/whitehall/blob/108623e34bfcfca22ed5b0d2b46a9ada2d102aba/app/validators/safe_html_validator.rb#L33-L43), so I suspect the [default `@attributes` setting](https://github.com/alphagov/whitehall/blob/108623e34bfcfca22ed5b0d2b46a9ada2d102aba/app/validators/safe_html_validator.rb#L15) doesn't do what we think it does i.e. nothing is being validated unless an explicit `attribute` or `attributes` keyword is passed ([example](https://github.com/alphagov/whitehall/blob/1b4391b328cd3b20afdb54622d9c62833cf7bb09/app/models/govspeak_content.rb#L5)). Out of scope for this PR, but something I intend to look into a bit later 👀